### ISSUE TITLE
Tworzenie oferty - 431 REQUEST_HEADER_FIELDS_TOO_LARGE and response bod

### ISSUE THREAD

**Issue by bmusialekcepd** [allegro/allegro-api#9964]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment
 - [ ] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

witam,

probuje zalozyc nowa oferte. W Wyniku dostaje blad:

Downloading image from URL 'https://media.drogerienatura.pl/catalog/product/5/9/5907747359338_intimelle_lagodzacy_plyn_500ml_1_44bf.png' failed. Server returned the following HTTP status code: 431 REQUEST_HEADER_FIELDS_TOO_LARGE and response body: ''. {"code":"DownloadError.RequestHeaderFieldsTooLarge","details":null,"path":"productSet[0].product.images","userMessage":"Downloading image from URL 'https://media.drogerienatura.pl/catalog/product/5/9/5907747359338_intimelle_lagodzacy_plyn_500ml_1_44bf.png' failed. Server returned the following HTTP status code: 431 REQUEST_HEADER_FIELDS_TOO_LARGE and response body: '

traceId: cedbb0152117cfe7

**Comment by MaciejFrackowiak** [comment]
Otrzymany przez Ciebie błąd sugeruje, że w requeście wysyłasz albo bardzo dużo nagłówków, albo jeden nagłówek jest za duży (np. referer albo cookies). Niestety, ale my nie logujemy dokładnie co to jest. [Tu](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/431) trochę więcej technicznych szczegółów.

**Comment by bmusialekcepd** [comment]
taki blad zwraca Panstwa serwer. To nie jest tak ze to podczas pobierania przez Was zdjecia otrzymujecie ten blad we wlasnym requescie i zwracanie nam?

dodatkowe przyklady trace-id:
cfd61fb5153aa1c2
19f0f52e6bd3b9b0
776fc28d804749c6

**Comment by MaciejFrackowiak** [comment]
Nie, błąd zwraca serwer, na którym znajdują się Twoje zdjęcia. Podczas próby ich ściągnięcia otrzymujemy właśnie błąd 431, a w rezultacie Twoje wywołanie dot. wystawienia oferty kończy się błędem 422.
